package.json

```diff
@@ -41,6 +42,9 @@
     "rimraf": "2.2.8"
   },
   "scripts": {
+    "build": "npm run build:client && npm run build:server",
+    "build:client": "cd client && brunch b",
```

Comment:
Maybe you can add some prebuild rules to update npm packages in client/ and update the locales from tx, like https://github.com/cozy/cozy-proxy/blob/master/package.json#L61-L62